### DISCUSSION TITLE
New design is sucks

### DISCUSSION THREAD

**Discussion by gggglglglg** [umami-software/umami#1983] [General]
I liked the design in v1.* so much
But what you did in version 2 sucks.

The top menu stretches the entire width of the screen. Have you tested your design on 27+, 32 inch monitors?

In version v1.* all the information was on the center, it was visually comfortable.

And the dark theme started to look terrible (colors)..

Bring back the old design, or make themes.

Thanks.

**Comment by MarcHagen** [comment]
> The top menu stretches the entire width of the screen. Have you tested your design on 27+, 32 inch monitors?

Yes agreeing on this as well. Makes not much sense, aleast make the buttons/links max width as the main container.

> And the dark theme started to look terrible (colors)..

And this is the really personal taste, the new dark theme is simple and basic.
